docs/package_info/glossary.rst

```diff
@@ -40,17 +34,6 @@ Glossary
 
   - ``t5`` These object represent :class:`.Subreddit`\ s.
 
-  .. _fullname_t6:
-
-  - ``t6`` These object represent ``Award``\ s, such as ``Reddit Gold`` or ``Reddit
-    Silver``.
```

Comment:
The API still returns [t6 objects](https://old.reddit.com/user/lil_spazjoekp/trophies.json).

Reply:
This should be changed to represent trophies.

Reply:
Good catch. Fixed.